sos/report/plugins/kubernetes.py

```diff
@@ -34,9 +34,22 @@ class Kubernetes(Plugin):
     ]
 
     kube_cmd = "kubectl"
+    resources = [
```

Comment:
To address 'R0912 too-many-branches' and 'R0914 too-many-locals' - essentially the function being too big.

Moved vars around to make the split easier and logical.